source/reactions.yaml

```diff
@@ -0,0 +1,111 @@
+  '/teams/{team_id}/channels/{channel_id}/posts/{post_id}/save_reaction':
+    post:
+      tags:
+        - posts
+      summary: React to a post
+      description: Saves a reaction made by the current user to the given post. Post reactions require Mattermost 3.6 or higher.

```

Comment:
Don't need to specify the Mattermost version, this will be taken care of shortly when I add separate pages for different API versions